java/display-kml/README.md

```diff
@@ -0,0 +1,34 @@
+# Display KML
+Display a KML layer from a URL, portal item, or local KML file.
+
+![Display KML App](display-kml.png)
+
+## How to use the sample
+Select the KML layer you'd like to display from the dropdown menu in the upper right
```

Comment:
Maybe you want to let the user know to download the file first and adb it to the device before they click on "KML from internal storage".  You can do something like this `Download file first, see [Offline Data](#offline-data)`

Reply:
Good idea!